ecs-cli/integ/README.md

```diff
@@ -0,0 +1,17 @@
+# Amazon ECS CLI - Integration Tests
+
+This directory contains tests intended to run against the production Amazon Elastic Container Service and other AWS Services. They are meant to test end-to-end functionality by executing commands as an ecs-cli user would.
```

Comment:
nit: "the production Amazon Elastic Container Service" sounds weird to me

Reply:
I took this phrasing from the Agent's Functional test README - good to highlight the fact these will be making actual calls to the real ECS (vs. mocks), I suppose, but agree the phrasing is a bit awkward. How about: "...intended to run against public Amazon ECS endpoints" ?